src/components/templates/Embeddable/Embeddable.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { EmbeddableVenue, Venue } from "types/venues";
+
+import "./Embeddable.scss";
+
+export interface EmbeddableProps {
+  venue: Venue;
+}
+
+export const Embeddable: React.FC<EmbeddableProps> = ({ venue }) => {
+  const typedVenue = venue as EmbeddableVenue;
+
+  if (!typedVenue?.iframeUrl) return <p>iframeUrl is missing</p>;
+
+  return (
+    <div className="embeddable" style={typedVenue.containerStyles}>
+      <iframe
+        title="embeddable-iframe"
+        src={typedVenue.iframeUrl}
+        className="embeddable__iframe"
+        style={typedVenue.iframeStyles}
+        {...typedVenue.iframeOptions}
```

Comment:
We should probably default to using our proper settings, and then allow them to be overridden by `typedVenue.iframeOptions`

```suggestion
      <iframe
        title="embeddable-iframe"
        src={typedVenue.iframeUrl}
        className="embeddable__iframe"
        style={typedVenue.iframeStyles}
        allow={IFRAME_ALLOW}
        allowFullScreen
        {...typedVenue.iframeOptions}
```

Reply:
For reference, this sets:

`src/settings.ts`
```
export const IFRAME_ALLOW =
  "accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture; fullscreen";
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1243/commits/9ba2d33cc99c0a5f51e657fa3f09be442ef9f48c